src/main/java/com/dougnoel/sentinel/steps/BaseSteps.java

```diff
@@ -172,4 +175,10 @@ public static void pressBrowserButton(String option) {
         }
     }
     
-}
\ No newline at end of file
+    @Then("^video recording (.*?)( was not)?(?: was)? attached to the report?$")
```

Comment:
elementName will never contain "not"
```suggestion
    @Then("^video recording was( not)? attached to the report?$")
```